samples/external/README.md

```diff
@@ -0,0 +1,28 @@
+# External Services
+
+Istio-enabled services are unable to access services and URLs outside of the cluster. Pods use <i>iptables</i> to transparently redirect all outbound traffic to the sidecar proxy, which only handles intra-cluster destinations.
```

Comment:
Should probably say "by default", esp with [this feature](https://deploy-preview-3140--preliminary-istio.netlify.com/docs/tasks/traffic-management/egress/#install-istio-with-access-to-all-external-services-by-default).

By default Istio-enabled services are unable ...